config/services.yaml

```diff
@@ -1,5 +1,5 @@
 settings:
-  kubeconfig: ~/config/kubeconfig.yaml
+  kubeconfig: /home/ptroc/.kube/config
```

Comment:
After short discussion I am fine with @ptroc suggestion, if do not do it for production later on.